content/en/real_user_monitoring/error_tracking/mobile/android.md

```diff
@@ -34,10 +34,30 @@ If you have not set up the Android SDK yet, follow the [in-app setup instruction
 
 1. Add the latest version of the [RUM Android SDK][4] to your Gradle dependencies.
 2. Configure your application's `env` and `variant` when [initializing the SDK][5].
-3. Run the Gradle task to upload your Proguard/R8 mapping file to Datadog in order to access deobfuscated stack traces. 
+3. Run the Gradle tasks to upload your Proguard/R8 mapping file and NDK symbol files to Datadog to access deobfuscated stack traces. 
 
 For any given error, you can access the file path, line number, and a code snippet for each frame of the related stack trace.
 
+### Add NDK crash reporting
+
+Your Android application may be running native code (C/C++) for performance or code reusability reasons. In order to get NDK crash reporting leverage the Datadog NDK plugin. 
+
+1. Declare `dd-sdk-android-ndk` as a dependency in your application module's build.gradle file.
```

Comment:
minor: in fact it can be in any module, requirement for the application module is enforced only for the Gradle plugin.